### PR TITLE
Climate control schedule commands

### PR DESCRIPTION
**Pull request by ben-voss**
Implements the climate control schedule commands.  Tested using a Gen5 Z-Stick and a POPP TRV Thermostat.

The climate control schedule commands allow up to 9 temperature setpoints per day to be configured in a device.  In my use case with the TRV this allows the TRV to run through the schedule independently without having to rely on the master controller to send it commands to change its setpoint several times a day. This saves a lot of battery power in the TRV as it avoids a lot of radio communication.

### CODE REVIEW

**Comment by ben-voss** [comment]
Sorry for the strange commit history I could not work out a better way to sync my fork and then apply the new changes on top.  All the changes in the first 2 commits are already in genielabs master.

**Comment by genemars** [comment]
Hi @ben-voss everything looks good. Would you mind removing me from the *Author* tag in the heading comments?

File: ZWaveLib/CommandClasses/ClimateControlSchedule.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            if (cmdType == (byte)Command.ScheduleReport) {
+                var climateControlScheduleValue = ClimateControlScheduleValue.Parse (message);
+                return new NodeEvent (node, EventParameter.ClimateControlSchedule, climateControlScheduleValue, 0);
+            }
+
+            if (cmdType == (byte)Command.ScheduleChangedReport) {
+                return new NodeEvent (node, EventParameter.ClimateControlScheduleChanged, message[2], 0);
+            }
+
+            if (cmdType == (byte)Command.ScheduleChangedReport) {
```
Comment (Bounz) [inline ZWaveLib/CommandClasses/ClimateControlSchedule.cs:52]: @ben-voss, I've just noted that you use `Command.ScheduleChangedReport` twice (here and on line 48). Looks like an error and should be `Command.ScheduleOverrideReport`.

Comment (ben-voss) [reply ZWaveLib/CommandClasses/ClimateControlSchedule.cs:52]: Thanks - good spot.  I will submit a pull request with a fix.

**Pull request merged**